templates/index.html

```diff
@@ -1,32 +1,7 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
-
-<!-- wp:query {"tagName":"main","layout":{"type":"constrained"}} -->
-<main class="wp-block-query">
-	<!-- wp:post-template -->
-		<!-- wp:group {"tagName":"article"} -->
-		<article class="wp-block-group">
-			<!-- wp:post-title {"isLink":true} /-->
-			<!-- wp:post-featured-image {"isLink":true} /-->
-			<!-- wp:post-excerpt /-->
-			<!-- wp:template-part {"slug":"post-meta"} /-->
-
-			<!-- wp:spacer {"height":40} -->
-			<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-			<!-- /wp:spacer -->
-		</article>
-		<!-- /wp:group -->
-	<!-- /wp:post-template -->
-
-	<!-- wp:group {"layout":{"type":"constrained"}} -->
-	<div class="wp-block-group">
-		<!-- wp:query-pagination -->
-			<!-- wp:query-pagination-previous /-->
-			<!-- wp:query-pagination-numbers /-->
-			<!-- wp:query-pagination-next /-->
-		<!-- /wp:query-pagination -->
-	</div>
-	<!-- /wp:group -->
+<!-- wp:group {"tagName":"main","align":"full","layout":{"type":"constrained"}} -->
+<main class="wp-block-group alignfull">
+	<!-- wp:pattern {"slug":"twentytwentyfour/posts"} /-->
```

Comment:
Because this pattern needs to have an H1 on the index, it means that other pages that use this pattern can end up with more than one H1.

Reply:
does that text need to be a heading at all?

Reply:
The index has to have an H1. I was told to use the pattern, so I needed to update that from H2 to H1.

Reply:
yeah, the text needs to be in a pattern anyway for translation. Let's go with this then